### PR TITLE
Fix check new versions bug if modoboa is offline

### PR DESCRIPTION
**Pull request by harobed**
Fix check new versions bug if modoboa is offline (example in train)

### CODE REVIEW

File: modoboa/core/extensions.py [hunk 1]
```diff
@@ -118,13 +118,14 @@ def load_all(self):
...
         result = []
+        new_extensions = {}
         if check_new_versions:
-            new_extensions = ModoAPIClient().list_extensions()
-            new_extensions = dict(
-                (ext["name"], ext["version"]) for ext in new_extensions
-            )
-        else:
-            new_extensions = {}
+            tmp = ModoAPIClient().list_extensions()
```
Comment (tonioo) [inline modoboa/core/extensions.py]: Could you rename tmp to something more verbose (like new_extensions as before) ?

Comment (harobed) [inline modoboa/core/extensions.py]: Ok

**Comment by harobed** [comment]
tmp variable removed.

**Comment by harobed** [comment]
Thanks

**Pull request merged**